packages/hash/shared/codegen.yml

```diff
@@ -16,14 +16,17 @@ generates:
         TextToken: ./types#TextToken
         Date: string
         UnknownEntityProperties: ./types#UnknownEntityProperties
+        VersionedUri: "@hashintel/hash-subgraph#VersionedUri"
         DataTypeWithoutId: "@hashintel/hash-shared/graphql/types#DataTypeWithoutId"
         DataTypeWithMetadata: "@hashintel/hash-subgraph#DataTypeWithMetadata"
         EntityTypeWithoutId: "@hashintel/hash-shared/graphql/types#EntityTypeWithoutId"
         EntityTypeWithMetadata: "@hashintel/hash-subgraph#EntityTypeWithMetadata"
         PropertyTypeWithoutId: "@hashintel/hash-shared/graphql/types#PropertyTypeWithoutId"
         PropertyTypeWithMetadata: "@hashintel/hash-subgraph#PropertyTypeWithMetadata"
+        EntityId: "@hashintel/hash-subgraph#EntityId"
+        EntityEditionId: "@hashintel/hash-subgraph#EntityEditionId"
         Edges: "@hashintel/hash-shared/graphql/types#Edges"
         Vertices: "@hashintel/hash-shared/graphql/types#Vertices"
     documents:
       - ./src/queries/*.ts
-      - ../api/src/collab/graphql/queries/*.ts
+      # - ../api/src/collab/graphql/queries/*.ts
```

Comment:
Nope, collab isn't used to any extent in the current main branch. I will remove the line instead of commenting this out to make that explicit. Collab is even removed from TS compilation in the API tsconfig.

Reply:
We should then move all GQL queries out of the `shared` package, right? If so perhaps this is too big for this PR, but we should probably leave a `@todo` and create a task.

Reply:
The queries related to page were moved from `collab` to `shared` when we switched to local page-saving, are you suggesting we move them from `shared` to `frontend`? They are only consumed by frontend currently, so that would be correct, but not sure if it should be done here or in one of the PRs coming up for fixing the frontend